dapper/mods/integration.py

```diff
@@ -29,38 +34,72 @@ def rk4(f, x, t, dt, order=4):
     dt : float
         Integration time step.
 
-    order : int, optional
-        The order of RK method. Default: 4
+    stages : int, optional
+        The number of stages of the RK method. Default: 4.  When stages=1, this becomes
+        Euler / Euler-Maruyama.
+
+    s : float
+        The diffusion coeffient for models with additive noise. Default: 0 for
+        deterministic integration.
 
     Returns
     -------
     ndarray
         State vector at the new time, `t+dt`
     """
-    if order >=1: k1 = dt * f(t     , x)                        # noqa
-    if order >=2: k2 = dt * f(t+dt/2, x+k1/2)                   # noqa
-    if order ==3: k3 = dt * f(t+dt  , x+k2*2-k1)                # noqa
-    if order ==4:                                               # noqa
-                  k3 = dt * f(t+dt/2, x+k2/2)                   # noqa
-                  k4 = dt * f(t+dt  , x+k3)                     # noqa
-    if    order ==1: return x + k1                              # noqa
-    elif  order ==2: return x + k2                              # noqa
-    elif  order ==3: return x + (k1 + 4*k2 + k3)/6              # noqa
-    elif  order ==4: return x + (k1 + 2*(k2 + k3) + k4)/6       # noqa
-    else: raise NotImplementedError                             # noqa
+
+    if s > 0.0:
```

Comment:
In this case s should generally be a float though, not int.  I don't know if it makes a difference in Python because of the type inference / coercion that occurs but this is just a habit from more static typing.

Reply:
I don't think there is good conformity in the code. But at least for python 3 it should not matter and I tend to lean to using just `0`.